Mathlib/Data/Matrix/Basic.lean

```diff
@@ -2098,7 +2098,6 @@ variants which this lemma would not apply to:
 * `Matrix.conjTranspose_intCast_smul`
 * `Matrix.conjTranspose_inv_natCast_smul`
 * `Matrix.conjTranspose_inv_intCast_smul`
-* `Matrix.conjTranspose_rat_smul`
```

Comment:
Can the `nsmul` and `zsmul` lemmas be eliminated in the same way? I think the history here is that there was previously no `Star` instance on these types.

Reply:
I already investigated this earlier today: No, because the `nsmul` and `zsmul` versions are true without commutativity of addition, but `Module` requires (by its definition) that addition is commutative.

Honestly, I would be happy to require commutativity of addition. Who cares about matrices on an non-comm group?!